ui/src/app/layout/dashboard/dashboard.component.ts

```diff
@@ -89,78 +89,106 @@ export class DashboardComponent implements OnInit {
-                            { topicName: topicName, topicLink: topicLink, appInfo: app });
-                    }
-
-                    return this.translate.stream('CHANGELOG_TOPIC_SUBSCRIBED_HTML',
-                        { topicName: topicName, topicLink: topicLink, appInfo: app });
-                }));
-        case 'TOPIC_UNSUBSCRIBED':
-            topicName = change.subscriptionMetadata.topicName;
-            topicLink = this.urlForRouterLink('/topics/' + topicName);
-            return this.applicationInfo(change.subscriptionMetadata.clientApplicationId).pipe(
-                flatMap(app => {
-                    if (!app) {
-                        return this.translate.stream('(unknown)').pipe(
-                            flatMap(s => this.translate.stream('CHANGELOG_TOPIC_UNSUBSCRIBED_HTML_NO_APP_LINK',
-                                { topicName: topicName, topicLink: topicLink, appInfo: { name: s } })));
-                    }
-                    if (!app.infoUrl) {
-                        return this.translate.stream('CHANGELOG_TOPIC_UNSUBSCRIBED_HTML_NO_APP_LINK',
-                            { topicName: topicName, topicLink: topicLink, appInfo: app });
-                    }
-                    return this.translate.stream('CHANGELOG_TOPIC_UNSUBSCRIBED_HTML',
-                        { topicName: topicName, topicLink: topicLink, appInfo: app });
-                }));
-        case 'TOPIC_PRODUCER_APPLICATION_ADDED':
-            topicName = change.topicName;
-            topicLink = this.urlForRouterLink('/topics/' + topicName);
-            const producerApplicationId = change.producerApplicationId;
-            return this.applicationInfo(producerApplicationId).pipe(mergeMap(
-                producer => this.translate.stream('CHANGELOG_PRODUCER_ADDED_HTML', {
+            case 'TOPIC_CREATED':
+                if (change.topicMetadata.type === 'INTERNAL') {
+                    return null;
+                }
+                topicName = change.topicMetadata.name;
+                topicLink = this.urlForRouterLink('/topics/' + topicName);
+                return this.translate.stream('CHANGELOG_TOPIC_CREATED_HTML', {
+                    topicName: topicName,
+                    topicLink: topicLink
+                });
+            case 'TOPIC_DELETED':
+                if (change.internalTopic) {
+                    return null;
+                }
+                topicName = change.topicName;
+                return this.translate.stream('CHANGELOG_TOPIC_DELETED_HTML', {topicName: topicName});
+            case 'TOPIC_DEPRECATED':
+
+                topicLink = this.urlForRouterLink('/topics/' + topicName);
+                topicName = change.topicName;
+                const eolDate = moment(change.eolDate).locale(this.translate.currentLang).format('L');
+                const americanDate = moment(change.eolDate).format('L');
```

Comment:
This is not Locale-safe (moment will use Locale of current user). And it is superfluous. Line 114 alone is enough, because currentLang always is also the language which will be used for the i18n lookup. So just remove americanDate here, and in the en.json.